Jenkinsfile

```diff
@@ -159,29 +160,12 @@ pipeline {
   }
   post {
     // This is the order that the methods are run. {always->success/abort/failure/unstable->cleanup}
-    always {
-      echo "always"
-    }
-    success {
-      echo "success"
-      script { lib_Status.set("SUCCESS", 'NRF', CI_STATE) }
-    }
-    aborted {
-      echo "aborted"
-      script { lib_Status.set("ABORTED", 'NRF', CI_STATE) }
-    }
-    unstable {
-      echo "unstable"
-      script { lib_Status.set("FAILURE", 'NRF', CI_STATE) }
-    }
-    failure {
-      echo "failure"
-      script { lib_Status.set("FAILURE", 'NRF', CI_STATE) }
-    }
-    cleanup {
-        echo "cleanup"
-        cleanWs()
-    }
+    always {   script { echo "always"; lib_Status.set( "${currentBuild.currentResult}" , 'TOP_LEVEL', CI_STATE) } }
+    // success  { }
```

Comment:
Are these meant to be filled in?

Reply:
I added a comment. 

no logic is needed here now, I left them to show they can be used and in what order they are called.